packages/integration-tests/src/entities.ts

```diff
@@ -75,3 +75,8 @@ export enum BlockExplorer {
   baseUrl = "http://localhost:3010",
   localNetwork = "/?network=local",
 }
+
+const txSumETH = "0.000009";
+export const Values = {
+  txSumETH: txSumETH,
```

Comment:
The reason why we need such contraction, instead of using just `export enum` is that we violate some strict type rules of TS. And received error, because type if txSumEth will be Values.

Reply:
Sorry, I might describe this issue in the wrong way.
Here's the example of full run of integration api tests on local machine:
We have this error present during running playbook scripts:
`345: Argument of type 'string' is not assignable to parameter of type 'Values'.`
<img width="1649" alt="image" src="https://github.com/matter-labs/block-explorer/assets/118197764/38e26576-d34d-4b01-9276-1e5d6aa2758c">

Reply:
This issue is fixed after new changes in integration-test-signed branch and currently working ok. 
```
export enum Values {
  txSumETH = "0.000009",
}
```

Reply:
Fixed this issue by adding type annotation to the functions that were failling because of type checks.
<img width="704" alt="image" src="https://github.com/matter-labs/block-explorer/assets/118197764/fa1973a5-add4-47a6-8054-a219573bda8d">